src/helpers/ComponentRegister.ts

```diff
@@ -27,6 +12,7 @@ export interface ComponentRegisterComponents {
    * Defaults to `"a"`.
    */
   Link?: LinkComponent;
+  LinkButton?: LinkButtonComponent;
```

Comment:
Docs please. Particularly, this could do with an explanation of why it's not the same as `Link`.